app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/ytdliability/further_return_check_tax_due.scala.html

```diff
@@ -40,10 +40,12 @@
     backLink: Call,
     yearToDateLiability: AmountInPence,
     previousYearToDateLiability: AmountInPence,
-    isATrust: Boolean
+    taxOwedOnOriginalReturn: Option[AmountInPence],
+    isATrust: Boolean,
+    isAmendReturn:Boolean
```

Comment:
you don't really need this flag - you can derive the value from `taxOwedOnOriginalReturn`. If it's defined, it's an amend return journey. Otherwise, it's not.